File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    w, h = 0, 0
    for size in sizes:
        if size[0] < size[1]:
            size[0], size[1] = size[1], size[0]
        w = max(w, size[0])
        h = max(h, size[1])
    return w * h

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int w, h, temp;
        w = 0; h = 0; temp = 0;
        for (int[] size : sizes) {
            if (size[0] < size[1]) {
                temp = size[0];
                size[0] = size[1];
                size[1] = temp;
            }
            w = Math.max(w, size[0]);
            h = Math.max(h, size[1]);
        }
        return w * h;
    }
}